.drone.star

```diff
@@ -1546,6 +1546,7 @@ def ocisServer(storage, accounts_hash_difficulty = 4):
     'KONNECTD_ISS': 'https://ocis-server:9200',
     'KONNECTD_TLS': 'true',
     'OCIS_LOG_LEVEL': 'warn',
+    'ACCOUNTS_DATA_PATH': '/var/tmp/ocis-accounts',
```

Comment:
This config variable is supposed to be not used anymore. There is one occurrence in `groups.DeleteGroup`, which doesn't make much sense...

Reply:
followup in #1061